### PR TITLE
Target .NET Standard 1.4 and .NET Framework 3.5

### PR DESCRIPTION
**Pull request by jingming**
This is a clone of https://github.com/twilio/twilio-csharp/pull/323

### CODE REVIEW

**Comment by dprothero** [comment]
Mad props to @galvesribeiro for getting us going in the right direction!

**Comment by galvesribeiro** [comment]
LGTM, but be aware that it will only test on `netstandard1.5` the way you did. The way I did on the other PR, test both. However, for whatever reason the `net35` installed on your CI isn't being detected. It build properly on my windows machine and all tests pass.

File: src/Twilio/Twilio.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (galvesribeiro) [inline src/Twilio/Twilio.csproj]: You don't need that in the latest tooling...

**Comment by jingming** [comment]
👍

**Pull request merged**